db/src/main/java/service/DBService.java

```diff
@@ -56,4 +56,9 @@ public Employee insertEmployee(Employee toSave) throws ClassNotFoundException, S
         }
         return null;
     }
+
+    public Employee updateEmployee(Employee toUpdate) throws ClassNotFoundException, SQLException {
+
+        String uEmployee = "update employees set birth_date='1997-09-04', first_name='Miguel' where emp_no = 500001";
```

Comment:
Ok, add missing fields to update

Reply:
```
String uEmployee = "update employees set birth_date = ?, first_name = ?, last_name = ?, hire_date = ? where emp_no = ?";
        Connection conn = null;
        String sql= "";
        PreparedStatement statement = conn.prepareStatement(sql);
        statement.setString(1, "1996-09-04");
        statement.setString(2, "Jorge");
        statement.setString(3, "Cardoso");
        statement.setString(4, "2020-04-09");
        statement.setString(5, `"500000");

        int rowsUpdated = statement.executeUpdate();
        if (rowsUpdated > 0) {
            System.out.println("An existing user was updated successfully!");
        }
        return toUpdate;
    }
```

Inventei um bocadinho, dá para fazer desta forma?

Reply:
Excellent. Keep up the good work

Reply:
Coloquei um código ainda por trabalhar, recomendações?